pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
       alsaLib nspr nss libnotify xorg.pixman yasm mesa
       xorg.libXScrnSaver xorg.scrnsaverproto pysqlite
       xorg.libXext xorg.xextproto sqlite unzip makeWrapper
-      hunspell libevent libstartup_notification cairo icu
+      hunspell libevent libstartup_notification cairo icu gconf
```

Comment:
Maybe pass `--disable-gconf` to prevent the gconf dependency? That's what we do with Firefox.